packages/server/src/instances/invalidator.ts

```diff
@@ -40,33 +36,33 @@ export class InstanceInvalidator {
     const conduit = (await this.conduits.get(conduitId))!
     const channel = this.channels.getById(conduit.channelId)
 
-    if (channel.initiable) {
+    if (channel.meta.initiable) {
       await this.instances.initialize(conduitId)
     }
 
-    if (!channel.lazy || !this.lazyLoadingEnabled) {
-      await this.instances.get(conduit.id)
+    if (!channel.meta.lazy || !this.lazyLoadingEnabled) {
+      await this.instances.start(conduit.id)
     }
   }
 
   private async onConduitDeleting(conduitId: uuid) {
-    this.cache.del(conduitId, true)
+    await this.instances.stop(conduitId)
   }
 
   private async onConduitUpdated(conduitId: uuid) {
-    this.cache.del(conduitId, true)
+    await this.instances.stop(conduitId)
```

Comment:
Notice here we were doing distributed invalidations on the cache. Now there is a dispatcher in the instance service that informs the other node when they need to stop their conduit